subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/store.py

```diff
@@ -846,6 +850,24 @@ def _parse_directory(fn_add_to_matrix, dirname, import_settings):
 
     print("parsing done :)")
 
+def _parse_start_end_times(dirname):
+    with open(dirname / '_ansible.log', 'rb') as file:
+        first = file.readline().decode('utf-8').strip()
+        file.seek(-20, 2)
+        while file.read(1) != b'\n':
+            file.seek(-2, 1)
+        last = file.readline().decode('utf-8').strip()
+        start_time = ' '.join(first.split(' ')[0:2]).split(',')[0]
+        end_time = ' '.join(last.split(' ')[0:2]).split(',')[0]
```

Comment:
can you add in a comment the string being parsed? it makes it easier to understand the parsing

here I think you want something like:
```
# first = "2023-04-14 17:19:19,808 p=770 u=psap-ci-runner n=ansible | ansible-playbook 2.9.27"
start_time = datetime.datetime.strptime(first.partition(",")[0], "%Y-%m-%d %H:%M:%S")
# last = 2023-04-14 17:25:31,697 p=770 u=psap-ci-runner n=ansible |...
end_time = start_time = datetime.datetime.strptime(last.partition(",")[0], "%Y-%m-%d %H:%M:%S")

return start_time, end_time
```